elrond-wasm/src/types/managed/managed_vec.rs

```diff
@@ -149,6 +149,13 @@ where
         });
     }
 
+    /// New `ManagedVec` instance with 1 element in it.
+    pub fn singleton(api: M, item: T) -> Self {
```

Comment:
I think the name `singleton` is misleading, since this word usually describes the singleton design pattern.
I would suggest a different name for this function (`from_item` / `from_single_item` or something along those lines).